docs/BR.md

```diff
@@ -1900,10 +1900,6 @@ The `subject` MUST comply with the requirements of [Section 7.1.4](#714-name-for
 
 ##### 7.1.2.2.3 Cross-Certified Subordinate CA Extensions
```

Comment:
Justification for below changes in this section:

```- In 7.1.2.2.3, the tables only differ in one row. I believe it would be beneficial to readers to make the first table apply to all Subordinate CA Certificates, and then follow it with a single-row table that overrides that one row for the case that the Subordinate CA is operated externally.```

Comment: I think this approach works. Some readers might prefer an explicit listing for each certificate type in an effort to avoid confusion, but I personally favor the new format.